docs/src/components/__tests__/Footer.test.js

```diff
@@ -109,48 +109,48 @@ describe('Footer', () => {
 		);
 	});
 
-	it('clicking primary link does not hide secondary links in nci big footer', async () => {
+	it('clicking Header does not hide secondary links in nci big footer', async () => {
 		render(<Footer variant="nci-big" />);
 
 		// First accordion activates
-		fireEvent.click(screen.getByText('Primary link 1'));
-		await waitFor(() => screen.getByLabelText('Primary link 1'));
-		expect(screen.getByLabelText('Primary link 1')).toHaveTextContent(
+		fireEvent.click(screen.getByRole('button', { name: 'Header 1' }));
+		await waitFor(() => screen.getByLabelText('Header 1'));
+		expect(screen.getByLabelText('Header 1')).toHaveTextContent(
 			'Secondary link 1'
 		);
 
 		// First accordion deactivates
-		fireEvent.click(screen.getByText('Primary link 1'));
-		await waitFor(() => screen.getByLabelText('Primary link 1'));
-		expect(screen.getByLabelText('Primary link 1')).toHaveTextContent(
+		fireEvent.click(screen.getByRole('button', { name: 'Header 1' }));
+		await waitFor(() => screen.getByLabelText('Header 1'));
+		expect(screen.getByLabelText('Header 1')).toHaveTextContent(
 			'Secondary link 1'
 		);
 
 		// Second accordion activates
```

Comment:
These following tests are probably overly redundant

Reply:
:+1: cleaned up tests for `nci-big` and removed script for `big `